keyboards/kbdfans/kbd67mkiirgb/kbd67mkiirgb.c

```diff
@@ -0,0 +1,116 @@
+#include "kbd67mkiirgb.h"
+const is31_led g_is31_leds[DRIVER_LED_TOTAL] = {
```

Comment:
This should be wrapped with an if-def.  That way, if rgb matrix is disabled, it doesn't break compliation. 

```suggestion
#ifdef RGB_MATRIX_ENABLE
const is31_led g_is31_leds[DRIVER_LED_TOTAL] = {
```